bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/TypeChecker.java

```diff
@@ -584,6 +584,13 @@ public static TypedescValue getTypedesc(Object value) {
      * @return the annotation value if present, nil else
      */
     public static Object getAnnotValue(TypedescValue typedescValue, String annotTag) {
+        MapValue annotMap = typedescValue.getAnnotations();
```

Comment:
We discussed to remove this method since it is no longer used. https://github.com/ballerina-platform/ballerina-lang/pull/36638/files#diff-331f52ed2675385bed30785e051a0db1b5a477c2e14ae43521fe51a3c5519f97L586 

Let's undo these changes.